examples/jf-admin-app/linux/README.md

```diff
@@ -9,16 +9,28 @@ An example application that acts as a Joint Fabric Administrator.
 
 <hr>
 
-## Building the Example Application on Linux
+## Prepare for building
+
+Before running any other build command, the `scripts/activate.sh` environment
+setup script should be sourced at the top level. This script takes care of
+downloading GN, ninja, and setting up a Python environment with libraries used
+to build and test.
+
+Run the following command at the top level:
+
+          $ source scripts/activate.sh
+
+## Checking if submodules are up to date
 
 -   Pull the required submodules
 
-          $ scripts/checkout_submodules.py --shallow --platform linux
+          $ ./scripts/checkout_submodules.py --shallow --platform linux
+
+## Building the Example Application on Linux
 
 -   Build the example application:
 
           $ cd ~/connectedhomeip/examples/jf-admin-app/linux
```

Comment:
The `source scripts/activate.sh` above assumes that we are in the `connectedhomeip` repo (which does not have to be in `$HOME`)

```suggestion
          $ cd examples/jf-admin-app/linux
```